src/__tests__/counterComment.test.js

```diff
@@ -0,0 +1,12 @@
+import countComment from "../counterComment";
+
+describe("testing count comment", () => {
+
+
+test("Counting comments", () => {
+   let commentList = [];
+   commentList.push("BA");
+    
+    expect(countComment(commentList)).toBe(1) 
+})
+})
```

Comment:
- [ ] Good job testing this count function, however, kindly add more tests to cover all possible edge cases for example when there are zero comments.